docs/elements.md

```diff
@@ -1865,6 +1866,37 @@ Note that only one of the attributes `source-file-name`, `submitted-file-name` o
 
 - [`pl-file-editor` to provide an in-browser code environment](#pl-file-editor-element)
 
+### `pl-card` element
+
+Displays question content inside a card component. Optionally displays content in a header, footer, or image caps via tag attributes.
+
+#### Sample element
+
+```html
+<pl-card header="Header" title="Title" width="50%" img-bottom-src="https://picsum.photos/536/354">
+  <pl-question-panel> This card is 50% width and has a bottom image cap. </pl-question-panel>
+</pl-card>
+```
+
```

Comment:
Any chance we could get an image here? I think the one from the PR would be perfect.

Reply:
I think that we're no longer including images in documentation anymore.

Reply:
Mmm, time for a new issues :)

Reply:
Added issue #6981